packages/bpk-component-grid-toggle/index.js

```diff
@@ -16,6 +16,6 @@
  * limitations under the License.
  */
 
-import component from './src/BpkGridToggle';
+import BpkGridToggle from './src/BpkGridToggle';
```

Comment:
Resolves error thrown in storybook. in conjunction with other change in grid-toggle.